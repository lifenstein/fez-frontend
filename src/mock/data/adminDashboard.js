export const adminDashboardConfig = {
    admin_users: [
        {id: 123, name: 'Michael Feeney'},
        {id: 23, name: 'Another Staff'},
        {id: 444, name: 'Elizabeth Alvey'},
        {id: 2333, name: 'Lee Sibbald'},
    ],
};

export const adminDashboardToday = {
    systemalerts: {
        total: 150,
        today: 25,
        assigned: 15,
        unassigned: 135,
        mine: 8,
    },
    works: {
        processed: 82,
        unprocessed: 15,
    },
    oa: {
        current: 256,
        total: 1256,
    },
};

export const adminDashboardQuickLinks =  [
    { qlk_id: 1, qlk_amount: 10, qlk_order: 1, qlk_title: '2021+ Imported Records with an Author ID and Research Subtypes Only', qlk_link: 'https://espace.library.uq.edu.au/records/search?searchQueryParams%5ball%5d=%28author_id:%5b1+TO+%2a%5d%29+AND+%28date:%5b2021%5c-01%5c-01T00%5c:00%5c:00Z+TO+2024%5c-12%5c-31T00%5c:00%5c:00Z+%5d%29+AND+%28subtype:%28%22Article%22%29+OR+subtype:%28%22Critical%22%29+OR+subtype:%28%22Fully%22%29+OR+subtype:%28%22Research+Book%22%29+OR+subtype:%28%22Research+Book+Chapter%22%29%29+AND+%28%28isi_loc:%5b%2a+TO+%2a%5d%29+OR+%28scopus_id:%5b%2a+TO+%2a%5d%29%29+AND+%28%28ismemberof:+UQ:639325%29+OR+%28ismemberof:+UQ:308300%29+OR+%28ismemberof:+UQ:180159%29+OR+%28ismemberof:+UQ:688533%29+OR+%28ismemberof:+UQ:727866%29%29&page=1&pageSize=20&sortBy=score&sortDirection=Desc' },
    { qlk_id: 7, qlk_amount: null, qlk_order: 6, qlk_title: 'UQ Library', qlk_link: 'https://www.library.uq.edu.au' },
    { qlk_id: 2, qlk_amount: 30, qlk_order: 7, qlk_title: '2022+ Imported Records with no Author ID with subtype exclusions', qlk_link: 'https://espace.library.uq.edu.au/records/search?searchQueryParams%5ball%5d=%28%21author_id:%5b1+TO+%2a%5d%29+AND+%28date:%5b2022%5c-01%5c-01T00%5c:00%5c:00Z+TO+2024%5c-12%5c-31T00%5c:00%5c:00Z%5d%29+AND+%28%21subtype:%28%22letter%22%29+OR+%21subtype:%28%22published+abstract%22%29+OR+%21subtype:%28%22review+of+book%22%29+OR+%21subtype:%28%22introduction%22%29+OR+%21subtype:%28%22poster%22%29%29+%28%28ismemberof:+UQ:639325%29+OR+%28ismemberof:+UQ:308300%29+OR+%28ismemberof:+UQ:180159%29+OR+%28ismemberof:+UQ:688533%29+OR+%28ismemberof:+UQ:727866%29%29&page=1&pageSize=20&sortBy=score&sortDirection=Desc' },
    { qlk_id: 3, qlk_amount: 60, qlk_order: 2, qlk_title: '2020 - 2023 Imported Records with an Author ID and Research Subtypes Only', qlk_link: 'https://espace.library.uq.edu.au/records/search?searchQueryParams%5ball%5d=%28author_id:%5b1+TO+%2a%5d%29+AND+%28date:%5b2020%5c-01%5c-01T00%5c:00%5c:00Z+TO+2023%5c-12%5c-31T00%5c:00%5c:00Z+%5d%29+AND+%28subtype:%28%22Article%22%29+OR+subtype:%28%22Critical%22%29+OR+subtype:%28%22Fully%22%29+OR+subtype:%28%22Research+Book%22%29+OR+subtype:%28%22Research+Book+Chapter%22%29%29+AND+%28%28isi_loc:%5b%2a+TO+%2a%5d%29+OR+%28scopus_id:%5b%2a+TO+%2a%5d%29%29+AND+%28%28ismemberof:+UQ:639325%29+OR+%28ismemberof:+UQ:308300%29+OR+%28ismemberof:+UQ:180159%29+OR+%28ismemberof:+UQ:688533%29+OR+%28ismemberof:+UQ:727866%29%29&page=1&pageSize=20&sortBy=score&sortDirection=Desc' },
    { qlk_id: 4, qlk_amount: 90, qlk_order: 3, qlk_title: '2021 - 2023 Imported Records with no Author ID with subtype exclusions', qlk_link: 'https://espace.library.uq.edu.au/records/search?searchQueryParams%5ball%5d=%28%21author_id:%5b1+TO+%2a%5d%29+AND+%28date:%5b2021%5c-01%5c-01T00%5c:00%5c:00Z+TO+2023%5c-12%5c-31T00%5c:00%5c:00Z%5d%29+AND+%28%21subtype:%28%22letter%22%29+OR+%21subtype:%28%22published+abstract%22%29+OR+%21subtype:%28%22review+of+book%22%29+OR+%21subtype:%28%22introduction%22%29+OR+%21subtype:%28%22poster%22%29%29+%28%28ismemberof:+UQ:639325%29+OR+%28ismemberof:+UQ:308300%29+OR+%28ismemberof:+UQ:180159%29+OR+%28ismemberof:+UQ:688533%29+OR+%28ismemberof:+UQ:727866%29%29+AND+%28%21institutional_status:%22Non-UQ%22%29&page=1&pageSize=20&sortBy=score&sortDirection=Desc' },
    { qlk_id: 5, qlk_amount: 110, qlk_order: 4, qlk_title: 'Not yet publicly available (with merged metadata)', qlk_link: 'https://espace.library.uq.edu.au/records/search?searchQueryParams%5ball%5d=%28isMemberOf:+UQ:676357%29+AND+%28%28doi:%5b%2a+TO+%2a%5d%29+OR+%28isi_loc:%5b%2a+TO+%2a%5d%29+OR+%28scopus_id:%5b%2a+TO+%2a%5d%29%29&page=1&pageSize=20&sortBy=score&sortDirection=Desc' },
    { qlk_id: 6, qlk_amount: 2345, qlk_order: 5, qlk_title: 'eSpace Followup', qlk_link: 'https://espace.library.uq.edu.au/records/search?page=1&pageSize=20&sortBy=score&sortDirection=Desc&searchQueryParams%5ball%5d%5bvalue%5d=ismemberof:+UQ:237927&searchQueryParams%5ball%5d%5blabel%5d=&searchMode=advanced' },
];

export const adminDashboardSystemAlerts = [
    {sat_id:1, sat_title: 'My Works - Claimed Work - UQ:8efd126 - uqwtomas', sat_content: 'User “Tomaszewski, Wojciech (Wojtek) (uqwtomas)” has indicated that they are the author “Tomaszewski, Wojciech (Wojtek) (87054)” on this work.', sat_link: 'https://espace.library.uq.edu.au/view/UQ:efd126', sat_created_date: '3/4/2024 15:55:00', sat_assigned_date: null, sat_resolved_date: undefined, sat_assigned_to: undefined, sat_resolved_by: undefined },
    {sat_id:12, sat_title: 'Issues on record - UQ:34555c6', sat_content: 'User “Tomaszewski, Wojciech (Wojtek) (uqwtomas)” has indicated that they are the author “Tomaszewski, Wojciech (Wojtek) (87054)” on this work.', sat_link: 'https://espace.library.uq.edu.au/view/UQ:efd126', sat_created_date: '3/4/2024 15:55:00', sat_assigned_date: null, sat_resolved_date: undefined, sat_assigned_to: 123, sat_resolved_by: undefined },
    {sat_id:13, sat_title: 'My Works - Claimed Work - UQ:1494946 - uqmdeben', sat_content: 'User “Tomaszewski, Wojciech (Wojtek) (uqwtomas)” has indicated that they are the author “Tomaszewski, Wojciech (Wojtek) (87054)” on this work.', sat_link: 'https://espace.library.uq.edu.au/view/UQ:efd126', sat_created_date: '3/4/2024 15:55:00', sat_assigned_date: null, sat_resolved_date: undefined, sat_assigned_to: 23, sat_resolved_by: undefined },
    {sat_id:14, sat_title: '(UQ eSpace) Keywords with full stops at the end', sat_content: 'User “Tomaszewski, Wojciech (Wojtek) (uqwtomas)” has indicated that they are the author “Tomaszewski, Wojciech (Wojtek) (87054)” on this work.', sat_link: 'https://espace.library.uq.edu.au/view/UQ:efd126', sat_created_date: '3/4/2024 15:55:00', sat_assigned_date: null, sat_resolved_date: undefined, sat_assigned_to: undefined, sat_resolved_by: undefined },
    {sat_id:51, sat_title: 'My Works - Disowned Work - UQ:4128e45 - uqsmcor3', sat_content: 'User “Tomaszewski, Wojciech (Wojtek) (uqwtomas)” has indicated that they are the author “Tomaszewski, Wojciech (Wojtek) (87054)” on this work.', sat_link: 'https://espace.library.uq.edu.au/view/UQ:efd126', sat_created_date: '3/4/2024 15:55:00', sat_assigned_date: null, sat_resolved_date: undefined, sat_assigned_to: 444, sat_resolved_by: undefined },
    {sat_id:16, sat_title: 'My Works - Claimed Work - UQ:e9a2910 - uqnpacha', sat_content: 'User “Tomaszewski, Wojciech (Wojtek) (uqwtomas)” has indicated that they are the author “Tomaszewski, Wojciech (Wojtek) (87054)” on this work.', sat_link: 'https://espace.library.uq.edu.au/view/UQ:efd126', sat_created_date: '3/4/2024 15:55:00', sat_assigned_date: null, sat_resolved_date: undefined, sat_assigned_to: undefined, sat_resolved_by: undefined },
    {sat_id:71, sat_title: 'My Works - Disowned Work - UQ:02daQbe - uqrelli6', sat_content: 'User “Tomaszewski, Wojciech (Wojtek) (uqwtomas)” has indicated that they are the author “Tomaszewski, Wojciech (Wojtek) (87054)” on this work.', sat_link: 'https://espace.library.uq.edu.au/view/UQ:efd126', sat_created_date: '3/4/2024 15:55:00', sat_assigned_date: null, sat_resolved_date: undefined, sat_assigned_to: undefined, sat_resolved_by: undefined },
    {sat_id:188, sat_title: 'My Works - Claimed Work - UQ:4489ca8 - uqwtomas', sat_content: 'User “Tomaszewski, Wojciech (Wojtek) (uqwtomas)” has indicated that they are the author “Tomaszewski, Wojciech (Wojtek) (87054)” on this work.', sat_link: 'https://espace.library.uq.edu.au/view/UQ:efd126', sat_created_date: '3/4/2024 15:55:00', sat_assigned_date: null, sat_resolved_date: undefined, sat_assigned_to: 2333, sat_resolved_by: undefined },
];

export const adminDashboardReportWorksData = [
    {
        id: 1,
        pid: 'UQ:abc123',
        date_created: '9/4/2024 10:44',
        work_type: 'Journal Article - Article (original research)',
        user: 'uqmfeene',
        topic: 'Updated record by O’Keefe, Christopher J, (User ID: 64567)',
    },
    {
        id: 2,
        pid: 'UQ:abc123',
        date_created: '9/4/2024 10:44',
        work_type: 'Journal Article - Article (original research)',
        user: 'uqmfeene',
        topic: 'Updated record by O’Keefe, Christopher J, (User ID: 64567)',
    },
    {
        id: 3,
        pid: 'UQ:abc123',
        date_created: '9/4/2024 10:44',
        work_type: 'Journal Article - Article (original research)',
        user: 'uqmfeene',
        topic: 'Updated record by O’Keefe, Christopher J, (User ID: 64567)',
    },
    {
        id: 4,
        pid: 'UQ:abc123',
        date_created: '9/4/2024 10:44',
        work_type: 'Journal Article - Article (original research)',
        user: 'uqmfeene',
        topic: 'Updated record by O’Keefe, Christopher J, (User ID: 64567)',
    },
    {
        id: 5,
        pid: 'UQ:abc123',
        date_created: '9/4/2024 10:44',
        work_type: 'Journal Article - Article (original research)',
        user: 'uqmfeene',
        topic: 'Updated record by O’Keefe, Christopher J, (User ID: 64567)',
    },
    {
        id: 6,
        pid: 'UQ:abc123',
        date_created: '9/4/2024 10:44',
        work_type: 'Journal Article - Article (original research)',
        user: 'uqmfeene',
        topic: 'Updated record by O’Keefe, Christopher J, (User ID: 64567)',
    },
];

export const adminDashboardReportSystemAlertsData = [
{
    id: 1,
    date_created: '9/4/2024 10:44',
    assigned_to: 'Michael Feeney',
    assigned_date: '9/4/2024 10:44',
    resolved_by: 'Elizabeth Alvey',
    resolved_date: '9/4/2024 10:44',
    title: 'My Works - Claimed Work - UQ:8efd126 - uqwtomas',
    content:
        'User “Tomaszewski, Wojciech (Wojtek) (uqwtomas)” has indicated that they are the author “Tomaszewski, Wojciech (Wojtek) (87054)” on this work.',
    link: 'https://espace.library.uq.edu.au/view/UQ:efd126',
},
{
    id: 2,
    date_created: '9/4/2024 10:44',
    assigned_to: 'Michael Feeney',
    assigned_date: '9/4/2024 10:44',
    resolved_by: 'Elizabeth Alvey',
    resolved_date: '9/4/2024 10:44',
    title: 'My Works - Claimed Work - UQ:8efd126 - uqwtomas',
    content:
        'User “Tomaszewski, Wojciech (Wojtek) (uqwtomas)” has indicated that they are the author “Tomaszewski, Wojciech (Wojtek) (87054)” on this work.',
    link: 'https://espace.library.uq.edu.au/view/UQ:efd126',
},
{
    id: 3,
    date_created: '9/4/2024 10:44',
    assigned_to: 'Michael Feeney',
    assigned_date: '9/4/2024 10:44',
    resolved_by: 'Elizabeth Alvey',
    resolved_date: '9/4/2024 10:44',
    title: 'My Works - Claimed Work - UQ:8efd126 - uqwtomas',
    content:
        'User “Tomaszewski, Wojciech (Wojtek) (uqwtomas)” has indicated that they are the author “Tomaszewski, Wojciech (Wojtek) (87054)” on this work.',
    link: 'https://espace.library.uq.edu.au/view/UQ:efd126',
},
{
    id: 4,
    date_created: '9/4/2024 10:44',
    assigned_to: 'Michael Feeney',
    assigned_date: '9/4/2024 10:44',
    resolved_by: 'Elizabeth Alvey',
    resolved_date: '9/4/2024 10:44',
    title: 'My Works - Claimed Work - UQ:8efd126 - uqwtomas',
    content:
        'User “Tomaszewski, Wojciech (Wojtek) (uqwtomas)” has indicated that they are the author “Tomaszewski, Wojciech (Wojtek) (87054)” on this work.',
    link: 'https://espace.library.uq.edu.au/view/UQ:efd126',
},
{
    id: 5,
    date_created: '9/4/2024 10:44',
    assigned_to: 'Michael Feeney',
    assigned_date: '9/4/2024 10:44',
    resolved_by: 'Elizabeth Alvey',
    resolved_date: '9/4/2024 10:44',
    title: 'My Works - Claimed Work - UQ:8efd126 - uqwtomas',
    content:
        'User “Tomaszewski, Wojciech (Wojtek) (uqwtomas)” has indicated that they are the author “Tomaszewski, Wojciech (Wojtek) (87054)” on this work.',
    link: 'https://espace.library.uq.edu.au/view/UQ:efd126',
},
{
    id: 6,
    date_created: '9/4/2024 10:44',
    assigned_to: 'Michael Feeney',
    assigned_date: '9/4/2024 10:44',
    resolved_by: 'Elizabeth Alvey',
    resolved_date: '9/4/2024 10:44',
    title: 'My Works - Claimed Work - UQ:8efd126 - uqwtomas',
    content:
        'User “Tomaszewski, Wojciech (Wojtek) (uqwtomas)” has indicated that they are the author “Tomaszewski, Wojciech (Wojtek) (87054)” on this work.',
    link: 'https://espace.library.uq.edu.au/view/UQ:efd126',
},
{
    id: 7,
    date_created: '9/4/2024 10:44',
    assigned_to: 'Michael Feeney',
    assigned_date: '9/4/2024 10:44',
    resolved_by: 'Elizabeth Alvey',
    resolved_date: '9/4/2024 10:44',
    title: 'My Works - Claimed Work - UQ:8efd126 - uqwtomas',
    content:
        'User “Tomaszewski, Wojciech (Wojtek) (uqwtomas)” has indicated that they are the author “Tomaszewski, Wojciech (Wojtek) (87054)” on this work.',
    link: 'https://espace.library.uq.edu.au/view/UQ:efd126',
}];