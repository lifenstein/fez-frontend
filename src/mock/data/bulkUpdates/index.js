export { default as bulkUpdatesList } from './bulkUpdatesList';