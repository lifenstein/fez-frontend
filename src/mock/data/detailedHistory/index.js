export { default as detailedHistory } from './detailedHistoryList';