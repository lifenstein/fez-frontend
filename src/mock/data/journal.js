export const journalDetails = {
    "data": {
        "jnl_jid": 8508,
        "jnl_title": "American Journal of Public Health",
        "jnl_publisher": "American Public Health Association",
        "jnl_created_date": "2020-11-16 07:48:52",
        "jnl_updated_date": "2021-01-21 03:33:04",
        "jnl_era_source_id": "13400",
        "fez_journal_abdc": {
            "jnl_abdc_for_code": 452815,
            "jnl_abdc_rating": "A*",
            "jnl_abdc_source_date": "2019-06-12",
            "jnl_abdc_title": "American Journal of Public Health",
            "jnl_abdc_issn": "0090-0036",
            "jnl_abdc_for_code_lookup": "1503 Business and Management"
        },
        "fez_journal_cite_score": {
            "jnl_cite_score_id": 25241,
            "jnl_cite_score_source_id": "19561",
            "jnl_cite_score_source_year": 2019,
            "jnl_cite_score_title": "American Journal of Public Health",
            "jnl_cite_score_publisher": "American Public Health Association",
            "jnl_cite_score_issn": "0090-0036",
            "fez_journal_cite_score_asjc_code": [
                {
                    "jnl_cite_score_asjc_code_id": 25241,
                    "jnl_cite_score_asjc_code_rank": 29,
                    "jnl_cite_score_asjc_code_rank_out_of": 516,
                    "jnl_cite_score_asjc_code_percentile": 94,
                    "jnl_cite_score_asjc_code_top_10_percent": true,
                    "jnl_cite_score_asjc_code": 453458,
                    "jnl_cite_score_asjc_code_cite_score": "6.6",
                    "jnl_cite_score_asjc_code_snip": "2.18",
                    "jnl_cite_score_asjc_code_sjr": "2.21",
                    "jnl_cite_score_asjc_code_percent_cited": "70.2",
                    "jnl_cite_score_asjc_code_order": 1,
                    "jnl_cite_score_asjc_code_quartile": 1,
                    "jnl_cite_score_asjc_code_lookup": "2739 Public Health, Environmental and Occupational Health"
                }
            ]
        },
        "fez_journal_doaj": {
            "jnl_doaj_jid": 12306,
            "jnl_doaj_source_id": "e05931389c0d41ad85a3bee708389841",
            "jnl_doaj_apc_average_price": 250,
            "jnl_doaj_apc_currency": 'AUD',
            "jnl_doaj_has_doi": true,
            "jnl_doaj_seal": false,
            "jnl_doaj_by": true,
            "jnl_doaj_nd": false,
            "jnl_doaj_nc": true,
            "jnl_doaj_sa": false,
            "jnl_doaj_homepage_url": "https://www.abdn.ac.uk/eitn/",
            "jnl_doaj_last_updated": "2020-12-07 16:17:55"
        },
        "fez_journal_era": [
            {
                "jnl_era_id": 73460,
                "jnl_era_source_id": "13400",
                "jnl_era_source_year": 2010,
                "jnl_era_title": "",
                "fez_journal_era_for_code": [
                    {
                        "jnl_era_for_code_id": 73460,
                        "jnl_era_for_code": "452650",
                        "jnl_era_for_code_order": 1,
                        "jnl_era_for_code_lookup": "1117 Public Health and Health Services"
                    }
                ]
            },
            {
                "jnl_era_id": 57795,
                "jnl_era_source_id": "13400",
                "jnl_era_source_year": 2012,
                "jnl_era_title": "",
                "fez_journal_era_for_code": [
                    {
                        "jnl_era_for_code_id": 57795,
                        "jnl_era_for_code": "452511",
                        "jnl_era_for_code_order": 1,
                        "jnl_era_for_code_lookup": "11 Medical and Health Sciences"
                    }
                ]
            },
            {
                "jnl_era_id": 28595,
                "jnl_era_source_id": "13400",
                "jnl_era_source_year": 2015,
                "jnl_era_title": "",
                "fez_journal_era_for_code": [
                    {
                        "jnl_era_for_code_id": 28595,
                        "jnl_era_for_code": "452511",
                        "jnl_era_for_code_order": 1,
                        "jnl_era_for_code_lookup": "11 Medical and Health Sciences"
                    }
                ]
            },
            {
                "jnl_era_id": 8508,
                "jnl_era_source_id": "13400",
                "jnl_era_source_year": 2018,
                "jnl_era_title": "American Journal of Public Health",
                "fez_journal_era_for_code": [
                    {
                        "jnl_era_for_code_id": 8508,
                        "jnl_era_for_code": "452511",
                        "jnl_era_for_code_order": 1,
                        "jnl_era_for_code_lookup": "11 Medical and Health Sciences"
                    }
                ]
            }
        ],
        "fez_journal_esi": [
            {
                "jnl_esi_subject": "SOCIAL SCIENCES, GENERAL",
                "jnl_esi_issn": "0090-0036",
                "jnl_esi_source_date": "2019-02-01",
                "jnl_esi_title": "AMERICAN JOURNAL OF PUBLIC HEALTH"
            },
            {
                "jnl_esi_subject": "SOCIAL SCIENCES, GENERAL",
                "jnl_esi_issn": "1541-0048",
                "jnl_esi_source_date": "2019-02-01",
                "jnl_esi_title": "AMERICAN JOURNAL OF PUBLIC HEALTH"
            }
        ],
        "fez_journal_historical_issn": [
            {
                "jnl_historical_issn_jid": 8508,
                "jnl_historical_issn": "0002-9572",
                "jnl_historical_issn_type": null,
                "jnl_historical_issn_order": 1
            },
            {
                "jnl_historical_issn_jid": 8508,
                "jnl_historical_issn": "1076-0704",
                "jnl_historical_issn_type": null,
                "jnl_historical_issn_order": 2
            },
            {
                "jnl_historical_issn_jid": 8508,
                "jnl_historical_issn": "1047-3491",
                "jnl_historical_issn_type": null,
                "jnl_historical_issn_order": 3
            },
            {
                "jnl_historical_issn_jid": 8508,
                "jnl_historical_issn": "1079-9567",
                "jnl_historical_issn_type": null,
                "jnl_historical_issn_order": 4
            },
            {
                "jnl_historical_issn_jid": 8508,
                "jnl_historical_issn": "1079-9559",
                "jnl_historical_issn_type": null,
                "jnl_historical_issn_order": 5
            },
            {
                "jnl_historical_issn_jid": 8508,
                "jnl_historical_issn": "0271-4353",
                "jnl_historical_issn_type": null,
                "jnl_historical_issn_order": 6
            },
            {
                "jnl_historical_issn_jid": 8508,
                "jnl_historical_issn": "0273-1975",
                "jnl_historical_issn_type": null,
                "jnl_historical_issn_order": 7
            },
            {
                "jnl_historical_issn_jid": 8508,
                "jnl_historical_issn": "0272-2313",
                "jnl_historical_issn_type": null,
                "jnl_historical_issn_order": 8
            },
            {
                "jnl_historical_issn_jid": 8508,
                "jnl_historical_issn": "0737-8769",
                "jnl_historical_issn_type": null,
                "jnl_historical_issn_order": 9
            }
        ],
        "fez_journal_issn": [
            {
                "jnl_issn_jid": 8508,
                "jnl_issn": "0090-0036",
                "jnl_issn_type": null,
                "jnl_issn_order": 1,
                "fez_sherpa_romeo": {
                    "srm_id": 3792,
                    "srm_issn": "0090-0036",
                    "srm_source_id": "10303",
                    "srm_journal_name": "American Journal of Public Health",
                    "srm_journal_link": "https:\/\/v2.sherpa.ac.uk\/id\/publication\/10303",
                    "srm_colour": "yellow",
                    "srm_date_updated": "2020-12-04 07:51:06"
                },
                "fez_ulrichs": {
                    "ulr_id": 650,
                    "ulr_issn": "0090-0036",
                    "ulr_status": "Active",
                    "ulr_source_id": "52095",
                    "ulr_title": "American Journal of Public Health",
                    "ulr_title_id": "41698",
                    "ulr_abbrev_title": "AJPH",
                    "ulr_publisher": "American Public Health Association",
                    "ulr_country": "United States",
                    "ulr_start_year": "1911",
                    "ulr_frequency": "Monthly",
                    "ulr_open_access": "0",
                    "ulr_open_access_url": null,
                    "ulr_reviewed": "1",
                    "ulr_refereed": "1",
                    "ulr_formats": "Print",
                    "ulr_description": "Contains reports of original research, demonstrations, evaluations, and other articles covering current aspects of public health.",
                    "ulr_last_updated": "2021-01-13 03:21:08"
                }
            },
            {
                "jnl_issn_jid": 8508,
                "jnl_issn": "1541-0048",
                "jnl_issn_type": null,
                "jnl_issn_order": 2,
                "fez_sherpa_romeo": {
                    "srm_id": 3793,
                    "srm_issn": "1541-0048",
                    "srm_source_id": "10303",
                    "srm_journal_name": "American Journal of Public Health",
                    "srm_journal_link": "https:\/\/v2.sherpa.ac.uk\/id\/publication\/10303",
                    "srm_colour": "yellow",
                    "srm_date_updated": "2020-12-04 17:57:15"
                },
                "fez_ulrichs": {
                    "ulr_id": 15274,
                    "ulr_issn": "1541-0048",
                    "ulr_status": "Active",
                    "ulr_source_id": "52108",
                    "ulr_title": "American Journal of Public Health",
                    "ulr_title_id": "41699",
                    "ulr_abbrev_title": null,
                    "ulr_publisher": "American Public Health Association",
                    "ulr_country": "United States",
                    "ulr_start_year": null,
                    "ulr_frequency": "Monthly",
                    "ulr_open_access": "0",
                    "ulr_open_access_url": null,
                    "ulr_reviewed": "0",
                    "ulr_refereed": "1",
                    "ulr_formats": "Online",
                    "ulr_description": "Contains reports of original research, demonstrations, evaluations, and other articles covering current aspects of public health.",
                    "ulr_last_updated": "2021-01-13 12:18:05"
                }
            }
        ],
        "fez_journal_jcr_scie": {
            "jnl_jcr_scie_id": 503,
            "jnl_jcr_scie_abbrev_title": "Am. J. Public Health",
            "jnl_jcr_scie_impact_factor": "5.381",
            "jnl_jcr_scie_5yr_impact_factor": "5.600",
            "jnl_jcr_scie_source_year": 2018,
            "jnl_jcr_scie_title": "AMERICAN JOURNAL OF PUBLIC HEALTH",
            "jnl_jcr_scie_issn": "0090-0036",
            "jnl_jcr_scie_start_year": 1911,
            "fez_journal_jcr_scie_category": [
                {
                    "jnl_jcr_scie_category_id": 503,
                    "jnl_jcr_scie_category_code": "NE",
                    "jnl_jcr_scie_category_description": "PUBLIC, ENVIRONMENTAL & OCCUPATIONAL HEALTH",
                    "jnl_jcr_scie_category_ranking": "12\/185",
                    "jnl_jcr_scie_category_quartile": "Q1",
                    "jnl_jcr_scie_category_order": 1
                },
                {
                    "jnl_jcr_scie_category_id": "2",
                    "jnl_jcr_scie_category_code": "AA",
                    "jnl_jcr_scie_category_description": "Category 2",
                    "jnl_jcr_scie_category_ranking": "45/459",
                    "jnl_jcr_scie_category_quartile": "Q3",
                    "jnl_jcr_scie_category_order": 2
                }
            ]
        },
        "fez_journal_jcr_ssci": {
            "jnl_jcr_ssci_id": 3085,
            "jnl_jcr_ssci_abbrev_title": "Am. J. Public Health",
            "jnl_jcr_ssci_impact_factor": "5.381",
            "jnl_jcr_ssci_5yr_impact_factor": "5.600",
            "jnl_jcr_ssci_source_year": 2018,
            "jnl_jcr_ssci_title": "AMERICAN JOURNAL OF PUBLIC HEALTH",
            "jnl_jcr_ssci_issn": "0090-0036",
            "jnl_jcr_ssci_start_year": 1911,
            "fez_journal_jcr_ssci_category": [
                {
                    "jnl_jcr_ssci_category_id": 3085,
                    "jnl_jcr_ssci_category_code": "NE",
                    "jnl_jcr_ssci_category_description": "PUBLIC, ENVIRONMENTAL & OCCUPATIONAL HEALTH",
                    "jnl_jcr_ssci_category_ranking": "6\/162",
                    "jnl_jcr_ssci_category_quartile": "Q1",
                    "jnl_jcr_ssci_category_order": 1
                }
            ]
        },
        "fez_journal_nature_index": {
            "jnl_nature_index_jid": 8508,
            "jnl_nature_index_source_date": "2020-11-12"
        },
        "fez_journal_pubmed": {
            "jnl_pubmed_jid": 8508,
            "jnl_pubmed_last_updated": "2020-12-15 05:42:00",
            "jnl_pubmed_source_id": "1254074"
        },
        "fez_journal_wos_category": [
                {
                "jnl_wos_category_title": "AMERICAN JOURNAL OF PUBLIC HEALTH",
                "jnl_wos_category": "Public, Environmental & Occupational Health",
                "jnl_wos_category_index": "SCIE",
                "jnl_wos_category_issn": "0090-0036",
                "jnl_wos_category_source_date": "2020-09-29",
                "fez_journal_cwts": {
                    "jnl_cwts_jid": 8508,
                    "jnl_cwts_source_year": "2020"
                }
            },
                {
                "jnl_wos_category_title": "AMERICAN JOURNAL OF PUBLIC HEALTH",
                "jnl_wos_category": "Public, Environmental & Occupational Health",
                "jnl_wos_category_index": "SSCI",
                "jnl_wos_category_issn": "0090-0036",
                "jnl_wos_category_source_date": "2020-09-29",
                "fez_journal_cwts": null
            },
            {
                "jnl_wos_category_title": "AMERICAN JOURNAL OF PUBLIC HEALTH",
                "jnl_wos_category": "Public, Environmental & Occupational Health",
                "jnl_wos_category_index": "SCIE",
                "jnl_wos_category_issn": "1541-0048",
                "jnl_wos_category_source_date": "2020-09-29",
                "fez_journal_cwts": null
            },
            {
                "jnl_wos_category_title": "AMERICAN JOURNAL OF PUBLIC HEALTH",
                "jnl_wos_category": "Public, Environmental & Occupational Health",
                "jnl_wos_category_index": "SSCI",
                "jnl_wos_category_issn": "1541-0048",
                "jnl_wos_category_source_date": "2020-09-29",
                "fez_journal_cwts": null
            }
        ]
    }
};