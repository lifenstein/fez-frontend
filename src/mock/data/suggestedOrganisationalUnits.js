export const suggestedOrganisationalUnits = {
    "data": [
        {
            "aut_id": 7624839,
            "org_title": "Information Systems and Resource Services (University of Queensland Library)",
            "org_id": 1248
        }
    ]
};