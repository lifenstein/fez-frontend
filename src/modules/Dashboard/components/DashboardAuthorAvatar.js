import React from 'react';
import PropTypes from 'prop-types';
import Avatar from '@material-ui/core/Avatar';
import locale from 'locale/pages';

const DashboardAuthorAvatar = ({ values }) => {
    const txt = locale.pages.dashboard.header.dashboardAuthorAvatar;
    return (
        <Avatar
            src={`https://its-ss-academicportal.s3.amazonaws.com/prod/uqresearchers/photo/thumbnail_${values.uqr_id}.jpg`}
            style={{
                backgroundSize: 'cover, cover',
                backgroundColor: 'rgba(0,0,0,0.3)',
                width: 125,
                height: 125,
                margin: 8,
            }}
            aria-label={`${txt.ariaPrefix} ${values.title} ${values.givenName} ${values.familyName}`}
            title={`${values.title} ${values.givenName} ${values.familyName}`}
        />
    );
};

DashboardAuthorAvatar.propTypes = {
    values: PropTypes.shape({
        uqr_id: PropTypes.any,
        title: PropTypes.string,
        familyName: PropTypes.string,
        givenName: PropTypes.string,
    }),
};

export default DashboardAuthorAvatar;
