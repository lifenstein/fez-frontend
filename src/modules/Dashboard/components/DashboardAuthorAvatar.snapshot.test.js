import React from 'react';
import DashboardAuthorAvatar from './DashboardAuthorAvatar';
import { authorDetails } from 'mock/data';
import { rtlRender } from 'test-utils';

function setup(testProps = {}) {
    // build full props list required by the component
    const props = {
        ...testProps,
        values: authorDetails.uqresearcher,
    };
    return rtlRender(<DashboardAuthorAvatar {...props} />);
}

describe('Dashboard Author Details test', () => {
    it('Render the authors details as expected for a UQ researcher)', () => {
        const { container } = setup();
        expect(container).toMatchSnapshot();
    });
});
