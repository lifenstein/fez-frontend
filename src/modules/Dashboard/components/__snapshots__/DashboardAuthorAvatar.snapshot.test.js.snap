// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Author Details test Render the authors details as expected for a UQ researcher) 1`] = `
<div>
  <div
    aria-label="Photograph of  Professor undefined undefined"
    class="MuiAvatar-root MuiAvatar-circular css-1wlk0hk-MuiAvatar-root"
    style="background-size: cover, cover; background-color: rgba(0, 0, 0, 0.3); width: 125px; height: 125px; margin: 8px;"
    title="Professor undefined undefined"
  >
    <img
      class="MuiAvatar-img css-1pqm26d-MuiAvatar-img"
      src="https://its-ss-academicportal.s3.amazonaws.com/prod/uqresearchers/photo/thumbnail_14.jpg"
    />
  </div>
</div>
`;
